src/content/contributing/community-membership/_index.en.md

```diff
@@ -0,0 +1,193 @@
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account
+- Have made multiple contributions to the project or community. Contribution
+  may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub
+    - Filing or commenting on issues on GitHub
+    - Contributing to community discussions (e.g. meetings, Slack, email
+      discussion forums, Stack Overflow)
+- Subscribed to [submariner-dev@googlegroups.com]
+- Have read the [contributor guide]
+- Actively contributing
+- Sponsored by 2 committers. **Note the following requirements for sponsors**:
+    - Sponsors must have close interactions with the prospective member - e.g.
+      code/design/proposal review, coordinating on issues, etc.
+    - Sponsors must be committers in at least 1 CODEOWNERS file either in any
+      repo in the [Submariner org]
+- [Open an issue][open issue] against the submariner-io/submariner repo
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of
+     your work on the project
+- Have your sponsoring committers reply confirmation of sponsorship: `+1`
+- Once your sponsors have responded, your request will be reviewed. Any missing
+  information will be requested.
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly
+  transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- They can be assigned to issues and PRs, and people can ask members for
+  reviews
+
+**Note:** Members who frequently contribute code are expected to proactively
+perform code reviews and work towards becoming a committer.
+
+## Committers
+
+Committers are able to review code for quality and correctness on some part of
+the project. They are knowledgeable about both the codebase and software
+engineering principles.
+
+**Until automation supports approvers vs reviewers:** They also review for
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** Entry in an CODEOWNERS file in a repo owned by the Submariner
+project.
+
+**Committer status is scoped to a part of the codebase.**
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a committer
+in an CODEOWNERS file:
+
+- Member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
```

Comment:
```suggestion
- Knowledgeable about the specific codebase
```
May be can be better explained that in my suggestion, what I try to say is, that it must be a knowledgeable person about the specific codebase we add permissions to.

Reply:
I think way the the intro is worded, "codebase" is already scoped to mean the "specific part of the codebase" you're a committer for.

One issue with that interpretation is the 5/20 review requirements would also apply to the narrowly scoped meaning of "codebase". In which case, I worry that there may be parts of the codebase we want to make someone a committer on for which there haven't been enough commits to review. If we run into that, maybe we're just trying to use too narrowly scoped CODEOWNERS rules. Or if we have small sets of files that need rules in many repos (like automation scripting), we can consider reviews to relevant files in all repos for the 5/20 count, not per-repo/rule.